packages/amplify-cli/src/extensions/amplify-helpers/read-json-file.js

```diff
@@ -1,4 +1,7 @@
 const fs = require('fs-extra');
+const os = require('os');
+
+const posPattern = /at position (\d+)$/;
```

Comment:
This seems a bit hard coded here, are we sure we'll always expect this kind of pattern?

Reply:
Implementation level: Yes

https://github.com/nodejs/node/blob/32e7e813e93ec3d0625d315658288769e448dd99/deps/v8/src/common/message-template.h#L424-L427
https://github.com/nodejs/node/blob/32e7e813e93ec3d0625d315658288769e448dd99/deps/v8/src/json/json-parser.cc#L251-L266

Specification level: No, I guess not.

I think providing line number and column number is ideal, but providing only filename is still good enough. If we want to show line and column seriously, we need to use or create custom JSON parser like https://www.npmjs.com/package/jsonlint.